account_avatax/models/account_move.py

```diff
@@ -287,24 +287,26 @@ def is_avatax_calculated(self):
         return self.is_sale_document()
 
     def _post(self, soft=True):
-        if self.is_avatax_calculated():
-            avatax_config = self.company_id.get_avatax_config_company()
-            if avatax_config and avatax_config.force_address_validation:
-                for addr in [self.partner_id, self.partner_shipping_id]:
-                    if not addr.date_validation:
-                        # The Validate action will be interrupted
-                        # if the address is not validated
-                        return addr.button_avatax_validate_address()
-            # We should compute taxes before validating the invoice
-            # to ensure correct account moves
-            # However, we can't save the invoice because it wasn't assigned a
-            # number yet
-            self.avatax_compute_taxes(commit=False)
+        for invoice in self:
+            if invoice.is_avatax_calculated():
+                avatax_config = self.company_id.get_avatax_config_company()
+                if avatax_config and avatax_config.force_address_validation:
+                    for addr in [self.partner_id, self.partner_shipping_id]:
+                        if not addr.date_validation:
+                            # The Validate action will be interrupted
+                            # if the address is not validated
+                            return addr.button_avatax_validate_address()
```

Comment:
@dreispt If the Avatax address validation is enabled and we use the bank statement to validate it, then the account.move will not be posted but the bank statement will be in posted state.

I think it is better to raise the Warning in this case instead of returning action

Reply:
OK for me.